Description: mime support, gcl dlopen support, emacs keybinding change
 Replace viewer commands with mime wrapper equivalents (e.g. 'see'),
 add :system-p t to gcl compiler flags to support possible dlopen fasl
 loading, remove maxima mode keybinding that interferes with standard
 emacs usage.  The first and third of these addressed long-closed
 debian bugs.
 .
 maxima (5.30.0-1) unstable; urgency=low
 .
   * New upstream release
Author: Camm Maguire <camm@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: maxima-5.45.1/interfaces/xmaxima/Tkmaxima/Constants.tcl
===================================================================
--- maxima-5.45.1.orig/interfaces/xmaxima/Tkmaxima/Constants.tcl
+++ maxima-5.45.1/interfaces/xmaxima/Tkmaxima/Constants.tcl
@@ -135,9 +135,9 @@ set maxima_priv(urlHandlers) {
     image/png  netmath
     image/jpg netmath
     image/jpeg netmath
-    application/postscript "ghostview -safer %s"
-    application/pdf "acroread %s"
-    application/x-dvi "xdvi %s"
+    application/postscript "see %s"
+    application/pdf "see %s"
+    application/x-dvi "see %s"
 }
 set maxima_priv(imagecounter) 0
 
Index: maxima-5.45.1/interfaces/emacs/emaxima/maxima.el
===================================================================
--- maxima-5.45.1.orig/interfaces/emacs/emaxima/maxima.el
+++ maxima-5.45.1/interfaces/emacs/emaxima/maxima.el
@@ -2489,7 +2489,7 @@ if completion is ambiguous."
     (define-key map "\C-c\)" 'maxima-check-parens)
     (define-key map [(control c) (control \))] 'maxima-check-form-parens)
 ;    (define-key map "\C-cC-\)" 'maxima-check-form-parens)
-    (define-key map "\177" 'backward-delete-char-untabify)
+;    (define-key map "\177" 'backward-delete-char-untabify)
     (setq maxima-mode-map map)))
 
 ;;;; Menu
@@ -3314,7 +3314,7 @@ To scroll through previous commands,
 (define-key inferior-maxima-mode-map [(meta control tab)] 
   'inferior-maxima-input-complete)
 (define-key inferior-maxima-mode-map "\e\t" 'inferior-maxima-complete)
-(define-key inferior-maxima-mode-map "\177" 'backward-delete-char-untabify)
+;(define-key inferior-maxima-mode-map "\177" 'backward-delete-char-untabify)
 (define-key inferior-maxima-mode-map "\C-c\C-k" 'maxima-stop)
 (define-key inferior-maxima-mode-map "\C-c\C-d" maxima-help-map)
 
Index: maxima-5.45.1/interfaces/emacs/misc/bookmode.el
===================================================================
--- maxima-5.45.1.orig/interfaces/emacs/misc/bookmode.el
+++ maxima-5.45.1/interfaces/emacs/misc/bookmode.el
@@ -680,7 +680,7 @@ file")
 		
 		  (and tem
 		       (list (concat "View "(nth 1 tem))
-			     'call-process  "ghostview" nil nil nil
+			     'call-process  "see" nil nil nil
 			     (expand-file-name(nth 1 tem))
 			     ))
 		  (list "Cancel")
@@ -705,7 +705,7 @@ file")
 			 (concat
 			  (cond ((looking-at "%PS") "")
 				(t "gzip -dc | "))
-			  "ghostview -")
+			  "see -")
 			 
 	)))
 	 
